data/json/npcs/Kindred/NPC_Brigitte_LaCroix.json

```diff
@@ -0,0 +1,505 @@
+    "responses": [
+      { "text": "What do you actually do with the bones?", "topic": "TALK_BONE_SEER_BONES" },
+      { "text": "Does anyone else share this belief?", "topic": "TALK_BONE_SEER_OTHERS" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_BONE_SEER_OTHERS",
+    "dynamic_line": "There are others who know the power the bones hold, but not all of them are my kin.  Not anymore, at least.",
+    "speaker_effect": { "effect": { "u_add_var": "knows_kindred", "type": "dialogue", "context": "kindred", "value": "yes" } },
+    "responses": [
+      { "text": "What do you mean 'not anymore'?", "topic": "TALK_BONE_SEER_COOPER" },
+      { "text": "Why do you gather these bones again?", "topic": "TALK_BONE_SEER_BONES" },
+      { "text": "Thank you for all the information, but I have to go now.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_BONE_SEER_COOPER",
+    "dynamic_line": "Some time ago, a man named Cooper joined me and my kin.  He showed great promise and devotion to our cause, but in his effort to weave songs, he justified increasingly grim actions.  We tried to stop him when we learned of this, but he managed to get away.  If you ever come across him, be sure to thread carefully, and if you can, stop him.",
+    "speaker_effect": { "effect": { "u_add_var": "knows_cooper", "type": "dialogue", "context": "kindred_cooper", "value": "yes" } },
+    "responses": [
+      {
+        "text": "Wait, Cooper?  Darren Cooper?  I've already met him.",
+        "topic": "TALK_BONE_SEER_METCOOPER2",
+        "condition": { "u_has_var": "talked_to_cooper", "type": "dialogue", "context": "first_meeting", "value": "yes" }
+      },
+      { "text": "What exactly did he do?", "topic": "TALK_BONE_SEER_COOPER2" },
+      { "text": "What cause is that?  What do you do with the bones?", "topic": "TALK_BONE_SEER_BONES" },
+      { "text": "I see.  I wanted to ask about something else…", "topic": "TALK_BONE_SEER" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_BONE_SEER_COOPER2",
+    "dynamic_line": "He began killing others to get their bones as fresh as possible.  Later, he…  he started carving the bones of the living.  Thought the anguish alone would empower the song.  The goal of the Kindred is to banish the shadow of the cataclysm, not to cast it.  Such heinous acts will likely twist the song of the world, but I don't think Cooper realises it.  To him, the end justifies any means.",
+    "speaker_effect": { "effect": { "u_add_var": "knows_cooper_kill", "type": "dialogue", "context": "kindred_cooper", "value": "yes" } },
+    "responses": [ { "text": "Well…  shit.  Maybe we should talk about something else.", "topic": "TALK_BONE_SEER" } ]
```

Comment:
You should consider the option for players to be non-good-aligned in their responses. It's up to you if you want to add it on this PR, but it should be possible for players to antagonize Brigitte and express admiration for cooper, for example.